ui/prometheus-plugin/src/components/PromQL.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import CodeMirror, { ReactCodeMirrorProps } from '@uiw/react-codemirror';
+import { PromQLExtension, CompleteConfiguration } from '@prometheus-io/codemirror-promql';
```

Comment:
Eventually, we need to fork this to get better support for template variables. Fortunately, syntax highlighting and linting (which is disabled) is the only thing that is funky right now with variables in certain places.